app/build.gradle

```diff
@@ -124,34 +123,34 @@ dependencies {
     implementation "org.jetbrains.kotlinx:kotlinx-coroutines-core:$kotlinCoroutinesVersion"
 
     // Android
-    implementation 'androidx.appcompat:appcompat:1.3.0'
-    implementation 'androidx.core:core-ktx:1.6.0'
-    implementation 'com.google.android.material:material:1.4.0'
+    implementation 'androidx.appcompat:appcompat:1.4.1'
+    implementation 'androidx.core:core-ktx:1.7.0'
+    implementation 'com.google.android.material:material:1.6.1'
     implementation 'androidx.cardview:cardview:1.0.0'
-    implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
-    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.3.1'
-    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.3.1'
+    implementation 'androidx.constraintlayout:constraintlayout:2.1.3'
+    implementation 'androidx.lifecycle:lifecycle-viewmodel-ktx:2.4.1'
+    implementation 'androidx.lifecycle:lifecycle-runtime-ktx:2.4.1'
     implementation 'androidx.security:security-crypto:1.1.0-alpha03'
-    implementation 'androidx.work:work-runtime-ktx:2.5.0'
+    implementation 'androidx.work:work-runtime-ktx:2.7.1'
 
     // Dependency Injection
-    api 'com.google.dagger:dagger:2.37'
-    kapt 'com.google.dagger:dagger-compiler:2.37'
+    api 'com.google.dagger:dagger:2.42'
+    kapt 'com.google.dagger:dagger-compiler:2.42'
 
     // Awala
     implementation 'tech.relaycorp:awala:1.65.0'
-    implementation 'tech.relaycorp:awala-keystore-file:1.6.0'
-    implementation 'tech.relaycorp:cogrpc:1.1.18'
-    implementation 'tech.relaycorp:cogrpc-okhttp:1.1.11'
+    implementation 'tech.relaycorp:awala-keystore-file:1.6.1'
+    implementation 'tech.relaycorp:cogrpc:1.1.21'
+    implementation 'tech.relaycorp:cogrpc-okhttp:1.1.12'
     testImplementation "tech.relaycorp:awala-testing:$awalaTestingVersion"
     androidTestImplementation "tech.relaycorp:awala-testing:$awalaTestingVersion"
 
     // Android TLS support for Netty
-    implementation "io.netty:netty-handler:4.1.65.Final"
+    implementation "io.netty:netty-handler:4.1.77.Final"
     implementation 'org.conscrypt:conscrypt-android:2.5.2'
 
     // Local and Internet-based Parcel Delivery Connections (PDCs)
```

Comment:
`    implementation "tech.relaycorp:doh:1.0.4"` needs ktor update as well to avoid conflicts